scripts/build/libc/glibc.sh

```diff
@@ -170,6 +170,9 @@ do_libc_backend() {
                              libc_target="${target}"
 
         CT_Popd
+        if [ "${multi_dir}" != "." ]; then
+            CT_EndStep
```

Comment:
As stated in a comment for #d8a3da, it might be better to change the existing `CT_DoStep` into a `CT_DoLog` instead.